.github/workflows/ci-frontend-a11y.yml

```diff
@@ -42,6 +42,16 @@ jobs:
       - name: Start server and log output
         run: npm run start &
 
+      - name: Start API Server for search results
+        run: |
+          cd ../api
+          make init db-seed-local start &
+          cd ..frontend
```

Comment:
do we need `cd ../frontend` with a slash - I think that's the only different b/w this workflow step and the one on `ci-frontend-e2e.yml`

Reply:
There is no functional difference between ..frontend and ../frontend. Up to you if you want that semantic change

Reply:
This was causing an issue and you were right. No difference in fish shell, likely a difference in our CI/CD pipeline. My mistake